docker-compose.yml

```diff
@@ -32,6 +32,16 @@ services:
       - "8000:80"
     depends_on:
       - app
-
+  queue:
+    image: rabbitmq:alpine
+    hostname: cloudiqueue
+    ports:
+      - "5672:5672"
+    environment:
+      - RABBITMQ_DEFAULT_USER=cloudigrade
+      - RABBITMQ_DEFAULT_PASS
```

Comment:
This is something that is used locally, and only in dev, therefore should we go ahead and set a default? I see no harm in making this `RABBITMQ_DEFAULT_PASS=cloudigrade`.

Reply:
Sounds good, if we want to keep it in dev mode I can just remove the default user and pass vars and it will default to `guest:guest`